src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -67,6 +67,7 @@
         <option value="BasicTestApp.SvgWithChildComponent">SVG with child component</option>
         <option value="BasicTestApp.TextOnlyComponent">Plain text</option>
         <option value="BasicTestApp.TouchEventComponent">Touch events</option>
+        <option value="BasicTestApp.ErrorComponent">Error throwing</option>
```

Comment:
Could you move this line to whatever point in the list keeps it all alphabetically sorted?